README.md

```diff
@@ -6,148 +6,106 @@
-
-Each exchange has one or more connectors in the Hummingbot codebase that supports a specific **market type** that the exchange supports:
-
- * **spot**: Connectors to central limit order book (CLOB) exchanges that trade spot markets
- * **perp**: Connectors to central limit order book (CLOB) exchanges that trade perpetual swap markets
- * **amm**: Connectors to decentralized exchanges that use the Automatic Market Maker (AMM) methodology
-
-Quarterly [Polls](https://docs.hummingbot.org/governance/polls/) allow HBOT holders decide how maintenance bandwidth and development bounties are allocated toward the connectors in the codebase.
-
-## Sponsors & Partners
-
-The Hummingbot Foundation, supported by its sponsors, partners and backers, is dedicated to fostering a robust, community-driven ecosystem for algorithmic crypto trading.
-
-### Sponsors
-
-- [Vega Protocol](https://vega.xyz/)
-- [Hyperliquid](https://hyperliquid.xyz/)
-- [CoinAlpha](https://coinalpha.com/)
-
-### Exchange Partners
-
-* [Binance Spot](https://www.binance.com/en/register?ref=FQQNNGCD) | [Binance Futures](https://www.binance.com/en/futures/ref?code=hummingbot)
-* [Kucoin](https://www.kucoin.com/ucenter/signup?rcode=272KvRf)
-* [Gate.io](https://www.gate.io/signup/5868285)
-* [AscendEx](https://ascendex.com/register?inviteCode=UEIXNXKW)
-* [Huobi](https://www.htx.com/)
-* [OKX](https://www.okx.com/join/1931920)
-
-For more information about the support provided by these partners, see the financial reports provided in [HBOT Tracker](https://docs.google.com/spreadsheets/d/1UNAumPMnXfsghAAXrfKkPGRH9QlC8k7Cu1FGQVL1t0M/edit#gid=285483484).
+* [Reported Volumes](https://p.datadoghq.com/sb/a96a744f5-a15479d77992ccba0d23aecfd4c87a52): Reported trading volumes across all Hummingbot instances
+* [Newsletter](https://hummingbot.substack.com): Get our newsletter whenever we ship a new release
+
+
+
+## Supported Exchange Connectors
+
+Hummingbot connectors standardize trading logic and order types across different types of exchanges:
+
+ * **CLOB Spot**: Connectors to spot markets on central limit order book (CLOB) exchanges
+ * **CLOB Perp**: Connectors to perpetual futures markets on CLOB exchanges
+ * **AMM**: Connectors to spot markets on Automatic Market Maker (AMM) decentralized exchanges
+
+### Partners and Sponsors
+
+We are grateful for the following exchange partners who support the development and maintenance of Hummingbot.
+
+| Connector ID | Exchange | Type | Docs | Discount |
+|----|------|-------|------|----------|
+| `binance` | [Binance](https://accounts.binance.com/register?ref=CBWO4LU6) | CLOB Spot | [Docs](https://hummingbot.org/exchanges/binance/) | [![Sign up for Binance using Hummingbot's referral link for a 10% discount!](https://img.shields.io/static/v1?label=Fee&message=%2d10%25&color=orange)](https://accounts.binance.com/register?ref=CBWO4LU6) |
```

Comment:
we have different referral link in docs, should we fix there as well?
https://hummingbot.org/exchanges/binance/#connector-info

Reply:
upd:
https://github.com/hummingbot/hummingbot-site/pull/405